Onboarding/onboarding-schedule-template.md

```diff
@@ -0,0 +1,43 @@
+# Ondboarding schedule template
+
+## Pre onboarding (1 week before)
+- Monday
+    - Send welcome email (this should include the intake questionaire and things they should expect going forward)
+    - Create a place to store all relavent documents for the team we are supporting
+    - Send invatations for onboarding meetings
+    - Send list of contact information about the support team
+- Tuesday
+    - Setup Slack accounts and invites
+    - Send instruction for setting up GitHub
+- Wednesday
+    - Connect GitHub account to any repos that are needed
+    - Setup ZenHub for team
+
+## Onboarding week
+- Monday
+    - Onboarding meeting/Kickoff meeting
+- Tuesday
```

Comment:
IMO all of the kickoff meetings should be on day 1.  Everyone won't be attending every meeting, and that way we get started with a clean slate of expectations and resources, and jump into day 2 with action on getting envs setup, and starting to work!

Reply:
Yeah that makes sense. I'll change it up to reflect that.